src/main/java/org/jenkinsci/main/modules/cli/auth/ssh/PublicKeySignatureWriter.java

```diff
@@ -45,12 +47,12 @@ public String asString(DSAPublicKey key) {
         try {
             ByteArrayOutputStream output = new ByteArrayOutputStream();
             DSAParams keyParams = Objects.requireNonNull(key.getParams(), "No DSA params available");
-            KeyEncodeHelper.encodeString(output, "ssh-dss", StandardCharsets.UTF_8);
+            KeyEncodeHelper.encodeString(output, "ssh-dss", StandardCharsets.ISO_8859_1);
```

Comment:
Is this valid for the future too?

Reply:
This is how the original code behaved before #10 . I do not have opinion about future, just fixing the known regression